Dockerfile

```diff
@@ -1,5 +1,5 @@
 # docker 17.06+ will let us do this:
-# ARG RUBY_VERSION=2.7
+# ARG RUBY_VERSION=3.0.2
```

Comment:
```suggestion
# ARG RUBY_VERSION=3.0
```

Reply:
I'm concerned a little about this change because it always takes the latest 3.0.X release.
At least one downside of it is in local development environment, developers will have to install new ruby version on their stations. WDYT?

Reply:
I think that path version change should not affect developer flow